package.json

```diff
@@ -66,37 +66,39 @@
     "@angular/platform-browser": "13.0.2",
     "@angular/platform-browser-dynamic": "13.0.2",
     "@angular/router": "13.0.2",
-    "@ng-bootstrap/ng-bootstrap": "^9.0.0",
+    "@ng-bootstrap/ng-bootstrap": "^10.0.0",
     "@ngrx/component-store": "13.0.1",
     "bootstrap": "^4.5.0",
     "rxjs": "^7.4.0",
     "tslib": "^2.0.0",
-    "zone.js": "0.11.4"
+    "zone.js": "0.11.4",
+    "@angular/compiler-cli": "13.0.2",
+    "@nrwl/cli": "13.2.1",
+    "@nrwl/nx-cloud": "12.5.4",
+    "@nrwl/workspace": "13.2.1",
+    "@angular-devkit/build-angular": "13.0.3",
+    "typescript": "4.4.4"
```

Comment:
These dependencies had to be moved here as they are required to build the prod version of webstore and admin-page